ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -33,9 +33,25 @@ const PersonalSignInfo: React.FC = () => {
   }
 
   const { from } = currentConfirmation.msgParams;
+  const isSiweSigReq = isSIWESignatureRequest(currentConfirmation);
 
   return (
     <>
+      {isSiweSigReq && (
+        <Box
```

Comment:
Given this general structure, including the `Box` and `ConfirmInfoRow`, is the same for the permit simulation, is it worth generalising that into a small component such as `SignatureSimulation` or `SimulationContainer`?

Reply:
Unlike Permit signature, this is pretty small thus I did not broke into separate component.

Reply:
Ref: https://github.com/airbnb/javascript/blob/master/react/README.md#basic-rules

Multi components per file is not a pattern I have seen being used much.

Reply:
In this case, I was suggesting we create a new component file since both SIWE and Permit simulations use the same structure.

Reply:
It would include the `ConfirmInfoRow` since the title and tooltip are also the same.

I'd agree if we had too many components without any re-use, but this would remove duplicated logic in two places so we could more easily maintain both.